synapse/config/registration.py

```diff
@@ -21,12 +21,26 @@
 
 
 class AccountValidityConfig(Config):
-    def __init__(self, config):
+    def __init__(self, config, synapse_config):
         self.enabled = (len(config) > 0)
```

Comment:
Sorry, can we have an explicit `enabled` field in the `account_validility` dict, which defaults to False?